uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/leave-type-wizard-form.component.js

```diff
@@ -12,7 +14,165 @@ define([], function () {
     }]
   };
 
-  function LeaveTypeWizardFormController ($log) {
+  function LeaveTypeWizardFormController ($log, sharedSettings) {
     $log.debug('Controller: LeaveTypeWizardFormController');
+
+    var vm = this;
+    var lineBreak = '\n\n';
+
+    vm.leaveTypeCategory = '';
+    vm.leaveTypeCategories = [
+      {
+        value: 'leave',
+        label: 'Leave',
+        icon: 'plane'
+      }
+    ];
+    vm.leaveTypeTitle = '';
+    vm.activeSettingsTabIndex = null;
+    vm.sections = [
+      {
+        name: 'general',
+        label: 'Leave Category'
+      },
+      {
+        name: 'settings',
+        label: 'Leave Category Settings'
+      }
+    ];
+    vm.sectionsTemplatesPath =
+      sharedSettings.sourcePath + 'leave-type-wizard/form/components/form-sections';
+    vm.settingsTabs = [
```

Comment:
I think both `vm.settingsTab` and `vm.sections` should be better placed in a `leave-type-wizard.constants` module, so you can separate the data from the business logic

Reply:
Done.